mobile/src/main/java/com/gelakinetic/mtgfam/fragments/TradeFragment.java

```diff
@@ -362,9 +367,9 @@ public void showDialog(final int id, final int sideForDialog, final int position
     /**
      * Sort the trades
      */
-    private void sortTrades(String sortOrder) {
+    private void sortTrades(String sortOrder, boolean dontSort) {
         /* If no sort type specified, return */
-        TradeComparator tradeComparator = new TradeComparator(sortOrder);
+        TradeComparator tradeComparator = new TradeComparator(sortOrder, dontSort);
         Collections.sort(mLeftList, tradeComparator);
         Collections.sort(mRightList, tradeComparator);
```

Comment:
If "sort by order" will be added to the hierarchy of sortable features, then it's going to remain a variable for each MtgCard. If I was keeping it as sort on/off, then the shallow copy would make sense.